src/amo/components/AddonDetail.js

```diff
@@ -146,7 +146,7 @@ export class AddonDetailBase extends React.Component {
           <h2 className="visually-hidden">
             {i18n.gettext('Extension Metadata')}
           </h2>
-          <AddonMeta />
+          <AddonMeta averageDailyUsers={addon.average_daily_users} />
```

Comment:
I think it's ok as a separate component because it connects to the `api.lang` state which pinpoints the exact place we'd need to do a re-render when that value changes. Plus, we will eventually put more metadata in there.